src/control/server/testdata/expect_daos_server_psm2.yml

```diff
@@ -15,12 +15,16 @@ servers:
   - CRT_TIMEOUT=30
   - CRT_CREDIT_EP_CTX=0
   scm_mount: /mnt/daos
+  scm_class: ram
+  scm_list: []
+  scm_size: 6
   bdev_class: nvme
   bdev_list:
   - 0000:81:00.0
   bdev_number: 0
   bdev_size: 0
   cliopts: []
+  formatcond: null
```

Comment:
What's formatcond here? Not finding it in the default conf file.

Reply:
it's the synchronisation condition, gets created during ControlService start-up in config.Servers and gets output when we dump structure to yaml. that's why it's present in the tests that verify config parsing because they work by dumping yaml

Reply:
Okay, got it.